build/ris.js

```diff
@@ -0,0 +1,401 @@
+'use strict'
+
+const https = require('https');
+const fs = require('fs');
+
+const { PassThrough, pipeline } = require('stream')
+const ndjson = require('ndjson')
+const { stringify } = require('qs');
+
+const bbox = { // should be approx. quadratic
+    north: 70.0,
+    west: -10.0,
+    south: 35.0,
+    east: 40.0,
+}
+
+const LIMIT_RESULTS = 10000;
+const PADDING_RATIO = 1.1;
+const MAX_DEPTH = 8;
+const MAX_REQUESTS = 1000;
+const TMP_PATH = 'tmp/';
+const STADA_FILE = 'stada.json'
+
+const DEFAULT_PRODUCTS = {
+    nationalExpress: false,
+    national: false,
+    regionalExpress: false,
+    regional: false,
+    suburban: false,
+    subway: false,
+    tram: false,
+    bus: false,
+    taxi: false,
+    ferry: false
+};
+
+const RIS_PRODUCTS_MAPPING = {
+    HIGH_SPEED_TRAIN: { product: 'nationalExpress', weight: 3 },
+    INTERCITY_TRAIN: { product: 'national', weight: 2 },
+    INTER_REGIONAL_TRAIN: { product: 'regionalExpress', weight: 1 },
+    REGIONAL_TRAIN: { product: 'regional', weight: 1 },
+    CITY_TRAIN: { product: 'suburban', weight: 1 },
+    SUBWAY: { product: 'subway', weight: 0.5 },
+    TRAM: { product: 'tram:', weight: 0.2 },
```

Comment:
```suggestion
    TRAM: { product: 'tram', weight: 0.2 },
```

This looks a little off